explainer.md

```diff
@@ -123,7 +123,7 @@ function BeginVRSession(isExclusive) {
 ```
 Once the session has started, some setup must be done to prepare for rendering.
 - A `VRFrameOfReference` must be created to define the coordinate system in which the `VRDevicePose` objects will be defined. See the Advanced Functionality section for more details about frames of reference.
-- The depth range of the session should be set to something appropriate for the application. This range will be used in the construction of the projection matricies provided by `VRPresentationFrame`.
+- The depth range of the session should be set to something appropriate for the application. This range will be used in the construction of the projection matricies provided by `VRFrame`.
```

Comment:
Fair point. I'll revert these for now. (I think Input might push them away from the `Presentation` verbiage again, though.)